fe4u-boilerplate-main/src/js/Lab2/generateSelectionOption.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+
+const CourseEnum = [
+    'Mathematics',
+    'Physics',
+    'English',
+    'Computer Science',
+    'Dancing',
+    'Chess',
+    'Biology',
+    'Chemistry',
+    'Law',
+    'Art',
+    'Medicine',
+    'Statistics',
+];
+const MonthEnum = [
```

Comment:
1. These constants should be in `contants.js1
2. The should be named in UPPERCASE:
```
export const COURSES = ...
export const MONTHES = ...
```